adapters/context_adapter.py

```diff
@@ -188,17 +190,19 @@ def _verify_in_node_operation(self):
         if self.type != NODE_INSTANCE:
             self._ctx.task.abort(
                 'ctx.node/ctx.instance can only be used in a {0} context but '
-                'used in a {1} context.'.format(NODE_INSTANCE, self.type))
+                'used in a {1} context.'.format(NODE_INSTANCE, self.type)
+            )
 
     def _verify_in_relationship_operation(self):
         if self.type != RELATIONSHIP_INSTANCE:
             self._ctx.task.abort(
                 'ctx.source/ctx.target can only be used in a {0} context but '
                 'used in a {1} context.'.format(RELATIONSHIP_INSTANCE,
-                                                self.type))
+                                                self.type)
+            )
 
 
-class _Blueprint(object):
+class BlueprintAdapter(object):
```

Comment:
consider adding a `_` to the prefix of any entity based context adapter (since they shouldn't be public)